scripts/zones/VeLugannon_Palace/mobs/Zipacna.lua

```diff
@@ -4,122 +4,355 @@
-    { x = -219, z = 420 },
-    { x = -257, y = 12, z = 417 },
-    { x = -261, y = 8, z = 400 },
-    { x = -256, y = 4, z = 381 },
-    { x = -238, y = 0, z = 380 },
-    { x = -214, z = 381 },
-    { x = -209, z = 387 },
-    { x = -202, z = 391 },
+    ['blueShortSideSpawnPath'] =
+    {
+        -- From blue side spawn point to the first blue door
+        { x = -195, y = 0,   z = 395 },
+        { x = -192, y = 0,   z = 390 },
+        { x = -189, y = 0,   z = 384 },
+        { x = -183, y = 0,   z = 380 },
+        { x = -165, y = 0,   z = 380 },
+        { x = -162, y = 0,   z = 380 },
+    },
+    ['blueBasementSpawnPath'] =
+    {
+        -- From the blue side basement to the first blue door
+        { x = -134, y = 16,  z = 380 },
+        { x = -150, y = 16,  z = 379 },
+        { x = -220, y = 16,  z = 379 },
+        { x = -220, y = 16,  z = 416 },
+        { x = -232, y = 16,  z = 419 },
+        { x = -240, y = 16,  z = 419 }, -- bottom of steps going back upstairs
+        { x = -255, y = 12,  z = 418 },
+        { x = -260, y = 11,  z = 412 },
+        { x = -260, y = 8,   z = 401 },
+        { x = -259, y = 5,   z = 388 },
+        { x = -255, y = 3,   z = 380 },
+        { x = -239, y = 0,   z = 380 },
+        { x = -215, y = 0,   z = 380 },
+        { x = -204, y = 0,   z = 389 },
+        { x = -197, y = 0,   z = 391 },
+        { x = -186, y = 0,   z = 382 },
+        { x = -177, y = 0,   z = 380 },
+        { x = -162, y = 0,   z = 380 }, --first blue door
+    },
+    ['yellowShortSideSpawnPath'] =
+    {
+        -- From the short yellow side spawn point to the first yellow door
+        { x = 191,  y = 0,   z = 399 },
+        { x = 192,  y = 0,   z = 389 },
+        { x = 186,  y = 0,   z = 381 },
+        { x = 182,  y = 0,   z = 379 },
+        { x = 174,  y = 0,   z = 379 },
+        { x = 161,  y = 0,   z = 379 },
+    },
+    ['yellowBasementSpawnPath'] =
+    {
+        -- From the yellow basement to the first yellow door
+        { x = 173,  y = 16,  z = 458 },
+        { x = 178,  y = 16,  z = 460 },
+        { x = 210,  y = 16,  z = 460 },
+        { x = 215,  y = 16,  z = 457 },
+        { x = 219,  y = 16,  z = 452 },
+        { x = 219,  y = 16,  z = 442 },
+        { x = 219,  y = 16,  z = 420 },
+        { x = 240,  y = 16,  z = 419 },
+        { x = 258,  y = 12,  z = 417 },
+        { x = 259,  y = 4,   z = 384 },
+        { x = 254,  y = 3,   z = 380 },
+        { x = 239,  y = 0,   z = 379 },
+        { x = 214,  y = 0,   z = 380 },
+        { x = 208,  y = 0,   z = 386 },
+        { x = 208,  y = 0,   z = 398 },
+        { x = 206,  y = 0,   z = 405 },
+        { x = 198,  y = 0,   z = 409 },
+        { x = 191,  y = 0,   z = 405 },
+        { x = 192,  y = 0,   z = 395 },
+        { x = 193,  y = 0,   z = 389 },
+        { x = 187,  y = 0,   z = 382 },
+        { x = 184,  y = 0,   z = 380 },
+        { x = 176,  y = 0,   z = 379 },
+        { x = 157,  y = 0,   z = 380 },
+    },
+    [1] =
```

Comment:
Why are you mixing string indexes for paths and numeric indexes for paths? Am I understanding that now these should all be indexed and looked up with their enum? `paths.BLUE_TO_BASEMENT` etc?

Reply:
Let me see what I can come up with on this.

Reply:
- Updated the `paths` enum to include the spawn point paths (thus allowing them to be referenced by their enum as well).
- Updated the `pathNodes` table to use the `paths` enums instead of hard-coded numbers.
- Adjusted the location of the spawn nodes in `pathNodes` to the end to match the order they appear in the `paths` enum.
- Re-Tested everything to ensure the next indices are working as expected.